src/models/Driver.ts

```diff
@@ -6,11 +6,11 @@ const driverSchema = new mongoose.Schema({
     required: true
   },
   phone_number: {
-    type: String,
+    type: Number,
```

Comment:
Phone number need to be string, RULE OF THUMB as long you don't need to do arithmetic operations on the number save it as string